advanced/Scripts/utils.sh

```diff
@@ -25,15 +28,48 @@
 #######################
 addOrEditKeyValPair() {
   local key="${1}"
-  local value="${2}"
-  local file="${3}"
-  if grep -q "^${key}=" "${file}"; then
-    sed -i "/^${key}=/c\\${key}=${value}" "${file}"
+  local value
+  local file
+  
+  # If two arguments have been passed, then the second one is the file - there is no value
+  if [ $# -lt 3 ]; then    
```

Comment:
```suggestion
  if [[ $# -lt 3 ]]; then    
```

Either double here or single on line 42, just so they match.